components/script/script_thread.rs

```diff
@@ -2453,10 +2724,14 @@ impl ScriptThread {
         }
     }
 
-    fn handle_resize(&self, id: PipelineId, size: WindowSizeData, size_type: WindowSizeType) {
+    /// Batch windo resize operations into a single "update the rendering" task,
+    /// or, if a load is in progress,
+    /// set the window size directly.
+    fn handle_resize_msg(&self, id: PipelineId, size: WindowSizeData, size_type: WindowSizeType) {
```

Comment:
This file uses both "msg" and "message" in function names, so let's standardize on "message":

```suggestion
    fn handle_resize_message(&self, id: PipelineId, size: WindowSizeData, size_type: WindowSizeType) {
```